sdcm/nemesis.py

```diff
@@ -3882,6 +3912,31 @@ def disrupt_run_unique_sequence(self):
         self.disrupt_grow_shrink_cluster()
         InfoEvent(message="Finished grow_shrink disruption").publish()
 
+    def disrupt_mv_delete_range(self):
```

Comment:
So eventually we do want to have this special nemesis?
I thought it's enough with the existing ones we have in case they happen in certain sequence.

Reply:
I think it's o.k. to leave this nemesis for a known issue, but at least to add the issue to the docstring